docs/docker/docker-images.md

```diff
@@ -0,0 +1,218 @@
+# Docker Images
+
+## Building your own image
+Please refer [Build your own image](https://docs.docker.com/mac/step_four/) for more reference.
+
+The below steps only provide instructions for creating and building docker images.
+
+### Build image using docker file
+The `docker build -t ng2-book-finder .` command takes the `Dockerfile` in the current directory, and builds an image called `ng2-book-finder` on your local machine.
+
+First Docker checks to make sure it has everything it needs to build.
+
+```
+ng2-book-finder (feature/docker-image) $ docker build -t ng2-book-finder .
```

Comment:
There is no more feature branch.. refer to master branch -  Check  https://github.com/introduction-angular2/ng2-book-finder